re #725: PR 5: Updated documentation. (#1473)
langchain4j/langchain4j: docs/docs/tutorials/5-ai-services.md

Now let's take a look at some examples.

### `boolean` as return type

```java
interface SentimentAnalyzer {

    @UserMessage("Does {{it}} has a positive sentiment?")
    boolean isPositive(String text);

}

SentimentAnalyzer sentimentAnalyzer = AiServices.create(SentimentAnalyzer.class, model);

boolean positive = sentimentAnalyzer.isPositive("It's awful!");
// false
```

### `Enum` as return type
```java
public enum TicketPriority {
    @Description("Critical issues such as payment gateway failures or security breaches.")
    CRITICAL,
    @Description("High-priority issues like major feature malfunctions or widespread outages.")
    HIGH,
    @Description("Low-priority issues such as minor bugs or cosmetic problems.")
    LOW
}

interface TicketAnalyzer {
    

>     @UserMessage("Analyzing the issue to determine its priority: |||{{it}}|||")

Reviewer: ```suggestion
    @UserMessage("Analyze the priority of the following issue: {{it}}")
```
